### PR TITLE
getSelectElementSize must return 1 on Android and iOS (fixes #303)

### LINKED ISSUE #303: Android and iOS ignore 'size'  and 'multiple' attributes for select elements
On mobile devices all select elements are displayed in one line, even if 'size' attribute is greater than 1, or element has 'multiple' attribute. So styleUtils.getSelectElementSize must always return 1 on these platforms.

### PR DESCRIPTION
**Pull request by AndreyBelym**
\cc @inikulin @churkin

### CODE REVIEW

File: src/client/utils/style.js [hunk 1]
```diff
@@ -149,12 +149,16 @@ export function getOptionHeight (select) {
...
 
-    return isIE && realSizeValue > childrenCount ?
+    return browserUtils.isIE && realSizeValue > childrenCount ?
            Math.round(selectScrollHeight / childrenCount) :
            Math.round(selectScrollHeight / Math.max(childrenCount, realSizeValue));
 }
 
 export function getSelectElementSize (select) {
+    if (browserUtils.isSafari && browserUtils.hasTouchEvents ||
+        browserUtils.isAndroid)
```
Comment (inikulin) [inline src/client/utils/style.js]: No need for the new line here. Add comment about why we should return `1` here.

**Comment by inikulin** [comment]
\r-

**Comment by inikulin** [comment]
lgtm

**Comment by inikulin** [comment]
ping @churkin

**Comment by churkin** [comment]
What about tests?

**Comment by AndreyBelym** [comment]
FPR
Also, can we bump version to 3.0.1 and publish it on npm after this PR, please?

File: test/client/fixtures/utils/style-test.js [hunk 2]
```diff
@@ -187,3 +188,46 @@ test('getOffset', function () {
     document.body.style.border            = '';
     document.documentElement.style.border = '';
 });
+
+test('getSelectElementSize', function () {
+    function createOption (parent, text) {
+        return $('<option></option>').text(text)
```
Comment (churkin) [inline test/client/fixtures/utils/style-test.js]: `$('<option></option>')` equal to  `$('<option>')`

File: src/client/utils/style.js [hunk 3]
```diff
@@ -149,12 +149,16 @@ export function getOptionHeight (select) {
...
         return getHeight(select);
 
-    return isIE && realSizeValue > childrenCount ?
+    return browserUtils.isIE && realSizeValue > childrenCount ?
            Math.round(selectScrollHeight / childrenCount) :
            Math.round(selectScrollHeight / Math.max(childrenCount, realSizeValue));
 }
 
 export function getSelectElementSize (select) {
+    //NOTE: iOS and Android ignore 'size' and 'multiple' attributes, all select elements behave like select with size=1
```
Comment (churkin) [inline src/client/utils/style.js]: `// NOTE:` 
Pls., add space  and dot at the end of comment.

File: test/client/fixtures/utils/style-test.js [hunk 4]
```diff
@@ -187,3 +188,46 @@ test('getOffset', function () {
...
+            .appendTo(parent);
+    }
+
+    function createSelect () {
+        var select = $('<select><select/>')
```
Comment (churkin) [inline test/client/fixtures/utils/style-test.js]: $('<select></select>') equal to $('<select>')

File: test/client/fixtures/utils/style-test.js (hunk 2 above)
Comment (inikulin) [inline test/client/fixtures/utils/style-test.js]: @churkin It's not a big deal. This auto-generated by editor auto-completion, so it's ok to keep it as is.

**Comment by churkin** [comment]
\r

**Comment by churkin** [comment]
@testcafe-build-bot \retest

**Comment by churkin** [comment]
@testcafe-build-bot \retest

**Comment by churkin** [comment]
lgtm

**Pull request merged**